packages/react-components/src/theming/icons.tsx

```diff
@@ -206,6 +205,16 @@ const DataLossPreventionProhibited16Regular = (): JSX.Element => {
   return <Prohibited16Regular />;
 };
 
+const HorizontalGalleryLeftButton = (): JSX.Element => {
+  const rtl = useTheme().rtl;
+  return rtl ? <ChevronRight20Regular /> : <ChevronLeft20Regular />;
+};
+
+const HorizontalGalleryRightButton = (): JSX.Element => {
+  const rtl = useTheme().rtl;
```

Comment:
Does this return a boolean?

Reply:
Yep! Really it should be called isRtl, or better `dir: 'ltr' | 'rtl`